mmdet/models/utils/res_layer.py

```diff
@@ -38,7 +38,7 @@ def __init__(self,
         if stride != 1 or inplanes != planes * block.expansion:
             downsample = []
             conv_stride = stride
-            if avg_down and stride != 1:
+            if avg_down:
```

Comment:
Does this modification influence other models like resnet or regnet?

Reply:
No. `avg_down` is used in `ResNetV1d` only, which doesn't have any checkpoint.